### PR TITLE
Ensure mcelog daemon is running and check it as client

### PR DESCRIPTION
**Pull request by ulyssesbbg**
Ref: https://github.com/blueboxgroup/ursula-monitoring/pull/79 for script that does the actual check

### CODE REVIEW

File: roles/common/tasks/monitoring.yml [hunk 1]
```diff
@@ -207,6 +207,11 @@
 - name: log scanning caching dir
   file: path=/var/cache/check-log owner=sensu state=directory
 
+- name: bad memory alert
+  sensu_check: name=bad-memory plugin=check-mcelog.sh use_sudo=true
+               auto_resolve=false interval=20 occurrences=1
+  notify: restart sensu-client
```
Comment (j2sol) [inline roles/common/tasks/monitoring.yml]: Wouldn't this need the same conditional around ppc64 as the installation of mcelog itself?

File: roles/common/tasks/system-tools.yml [hunk 2]
```diff
@@ -49,6 +49,10 @@
   apt: pkg=mcelog
   when: common.system_tools.mcelog|bool and ansible_architecture != "ppc64le"
 
+- name: mcelog service enabled
+  service: name=mcelog state=started enabled=true
+  when: common.system_tools.mcelog|bool and ansible_architecture != "ppc64le"
```
Comment (j2sol) [inline roles/common/tasks/system-tools.yml]: I wonder if we'll need a log rotation entry for this too.

File: roles/common/tasks/monitoring.yml [hunk 3]
```diff
@@ -207,6 +207,13 @@
 - name: log scanning caching dir
   file: path=/var/cache/check-log owner=sensu state=directory
 
+- name: bad memory alert
+  sensu_check: name=bad-memory plugin=check-mcelog.sh use_sudo=true
+               auto_resolve=false interval=20 occurrences=1
+               args='--criticality'
```
Comment (j2sol) [inline roles/common/tasks/monitoring.yml]: I think the intent is that a value would come along with the --criticality. For this PR you can avoid sending that argument, we just want the check itself to accept such an argument if it were provided.

Comment (paulczar) [inline roles/common/tasks/monitoring.yml]: correct `--criticality {{ common.monitoring.sensu_checks.bad_memory.criticality }}`  or whatever the correct var path is.   should be set to warning or critical.  there's examples of others in the repo doing this.

Comment (j2sol) [inline roles/common/tasks/monitoring.yml]: This still needs updating @ulyssesbbg Will you be able to update it in time for our 3.1.0 release?

**Comment by paulczar** [comment]
closing all PRs with no recent activity.

**Pull request closed without merging**